src/lib/components/SearchBar/SearchBarILS.js

```diff
@@ -65,13 +55,6 @@ export class SearchBarILS extends Component {
         size="big"
         placeholder={placeholder}
         className={`${parentClass} ils-searchbar`}
-        ref={(input) => {
```

Comment:
are you sure we want to remove the focus from this component which is widely used across the whole application ?